packages/app/src/components/nominationBanner/nominationBanner.js

```diff
@@ -1,14 +1,25 @@
 import style from './style.css'
-import React from 'react';
+import React, { useState, setState } from 'react';
 const states = require('us-state-codes');
 
+const [hasBeenClicked, setHasBeenClicked] = useState(false);
+
 const NominationBanner = ({ nomination }) => {
   const date = new Date(nomination.dateReceived).toLocaleDateString()
   const lastName = nomination.patientName ? nomination.patientName.split(' ')[1] : ''
   const state = states.getStateCodeByStateName(nomination.hospitalState)
   const nominationName = `${lastName}-${state}`
   const formattedAmount = nomination.amountRequestedCents ? (nomination.amountRequestedCents).toFixed(2).replace(/\d(?=(\d{3})+\.)/g, '$&,') : ''
 
+  function handleClick(){
+    if (hasBeenClicked === false) {
```

Comment:
When you call `useState`, you're creating a React hook. This isn't how hooks are used so take a look at the docs.